pyswarms/backend/operators.py

```diff
@@ -207,3 +208,35 @@ def compute_position(swarm, bounds, bh):
         raise
     else:
         return position
+
+
+def compute_objective_function(swarm, objective_func, pool=None, **kwargs):
+    """Evaluate particles using the objective function
+
+    This method evaluates each particle in the swarm according to the objective function passed.
+
+    If a pool is passed, then the evaluation of the particles is done in parallel using multiple processes.
+
+    Note: due to pickling limitations, if the pyswarms.cost decorator is used to build the objective function,
```

Comment:
@whzup I have this docstring here to highlight the incompability of multiprocessing and the `cost` decorator. Should I also add this docstring to `cost` decorator module?

Reply:
Would it be a lot of work to just fix the decorator such that it works? Seems there are some solutions in the links you provided above.

Reply:
I tried using those solutions before but I couldn't make it work...

Today, I tried again and it worked 🤣 I think I was forgetting to put the `cost_func` argument inside the `@wraps` decorator from functools.

I've updated the PR to contain this fix 👍